src/makeRobot.js

```diff
@@ -38,7 +38,68 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const rescueCenter = {
+    x: 1400,
+    y: 500,
+  };
+
+  const robot = {
+    name,
+    wheels,
+    version,
+    coords: {
+      x: 0,
+      y: 0,
+    },
+
+    get info() {
+      return `name: ${this.name}, `
+       + `chip version: ${this.version}, wheels: ${this.wheels}`;
```

Comment:
```suggestion
      return `name: ${this.name}, `
       + `chip version: ${this.version}, `
       + `wheels: ${this.wheels}`;
```
As a variant, to make everything in one style